### PR TITLE
Ui fix in user profile

### PR DESCRIPTION
**Pull request by ace0625**
- Fix line padding in user profile.
- Fix margin in expertise section.

### CODE REVIEW

File: src/components/userProfile/UserProfile.js [hunk 1]
```diff
@@ -43,7 +44,25 @@ class UserProfile extends Component {
...
+      onStartShouldSetPanResponder: () => true,
+      onPanResponderMove: Animated.event([null, {
+        dy: this.state.pan.y,
+      },
+      ]),
+
+      onPanResponderRelease: (e, gesture) => {
+          Animated.spring(
+              this.state.pan.y = 0,
+              // { toValue: { x: 0, y: 0 } },
```
Comment (ifndefdeadmau5) [inline src/components/userProfile/UserProfile.js:61]: I think you forgot to delete this??

File: src/components/userProfile/UserProfile.js [hunk 2]
```diff
@@ -201,6 +221,14 @@ class UserProfile extends Component {
     );
   }
 
+  handleScroll(event) {
+    if (event.nativeEvent.contentOffset.y < 0) {
+      this.setState({
+        scale: event.nativeEvent.contentOffset.y / -200 + 1,
```
Comment (ifndefdeadmau5) [inline src/components/userProfile/UserProfile.js:227]: Unexpected mix of '-' and '*'

@MechanicKim Suggested applying this new rule
https://github.com/sv-bootcamp/bridgeme-client/pull/143#discussion-diff-93888737R54

File: src/components/userProfile/UserProfile.js [hunk 3]
```diff
@@ -43,7 +44,25 @@ class UserProfile extends Component {
...
+
+    this.panResponder = PanResponder.create({
+      onStartShouldSetPanResponder: () => true,
+      onPanResponderMove: Animated.event([null, {
+        dy: this.state.pan.y,
+      },
+      ]),
+
+      onPanResponderRelease: (e, gesture) => {
+          Animated.spring(
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:59]: Expected indentation of 8 spaces but found 10

File: src/components/userProfile/UserProfile.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ import {
   Dimensions,
   Image,
   ListView,
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:8]: 'ListView' is defined but never used

File: src/components/userProfile/UserProfile.js [hunk 5]
```diff
@@ -43,7 +44,25 @@ class UserProfile extends Component {
...
+      },
+      ]),
+
+      onPanResponderRelease: (e, gesture) => {
+          Animated.spring(
+              this.state.pan.y = 0,
+              // { toValue: { x: 0, y: 0 } },
+              this.state.scale = 1,
+          ).start();
+        },
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:64]: Expected indentation of 6 spaces but found 8

File: src/components/userProfile/UserProfile.js [hunk 6]
```diff
@@ -84,7 +103,8 @@ class UserProfile extends Component {
   getProfileImage(status) {
     let image;
     if (status.profile_picture) {
-      image = { uri: status.profile_picture_large ? status.profile_picture_large : status.profile_picture };
+      image = { uri: status.profile_picture_large ?
+          status.profile_picture_large : status.profile_picture, };
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:107]: Unexpected trailing comma

File: src/components/userProfile/UserProfile.js [hunk 7]
```diff
@@ -84,7 +103,8 @@ class UserProfile extends Component {
...
       return image;
     } else {
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:109]: Unnecessary 'else' after 'return'

File: src/components/userProfile/UserProfile.js [hunk 8]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
       );
     }
 
+    let { scale } = this.state;
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:307]: 'scale' is never reassigned. Use 'const' instead

File: src/components/userProfile/UserProfile.js [hunk 9]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
+    let imageStyle = { transform: [{ scale }] };
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:308]: 'imageStyle' is never reassigned. Use 'const' instead

File: src/components/userProfile/UserProfile.js [hunk 10]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
+
     return (
       <View style={{
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:311]: Property should be placed on a new line

File: src/components/userProfile/UserProfile.js [hunk 11]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
         justifyContent: 'space-between',
         flexDirection: 'column',
         flex: 1,
       }}>
-        <ScrollView>
+        <ScrollView
+            scrollEventThrottle={16}
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:317]: line 317, 318
Expected indentation of 10 space characters but found 12

File: src/components/userProfile/UserProfile.js [hunk 12]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
 
+    let { scale } = this.state;
+    let imageStyle = { transform: [{ scale }] };
+
     return (
       <View style={{
         justifyContent: 'space-between',
         flexDirection: 'column',
         flex: 1,
       }}>
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:315]: The closing bracket must be aligned with the line containing the opening tag (expected column 7 on the next line)

File: src/components/userProfile/UserProfile.js [hunk 13]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
-        <ScrollView>
+        <ScrollView
+            scrollEventThrottle={16}
+            onScroll={this.handleScroll.bind(this)}>
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:318]: The closing bracket must be aligned with the line containing the opening tag (expected column 9 on the next line)

File: src/components/userProfile/UserProfile.js [hunk 14]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
           <StatusBar
             backgroundColor = "transparent"
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:320]: line 320, 321
There should be no space after(or before) '='

File: src/components/userProfile/UserProfile.js [hunk 15]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
           />
-          <LinearGradient style={styles.profileImgGradient} start={[0.0, 0.25]} end={[0.5, 1.0]}
-            colors={['#546979', '#08233a']}>
-            <Image style={styles.profileImage}
-              source={this.state.profileImage} />
-          </LinearGradient>
-          <Image style={styles.bookmarkIcon}
-            source={require('../../resources/icon-bookmark.png')}/>
+          <Animated.View
+              {...this.panResponder.panHandlers}
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:325]: line 325, 326
Expected indentation of 12 space characters but found 14

File: src/components/userProfile/UserProfile.js [hunk 16]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
+              style={[this.state.pan.getLayout(), imageStyle]}>
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:326]: The closing bracket must be aligned with the line containing the opening tag (expected column 11 on the next line)

File: src/components/userProfile/UserProfile.js [hunk 17]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
+            <LinearGradient
+                style={styles.profileImgGradient} start={[0.0, 0.25]} end={[0.5, 1.0]}
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:328]: line 328, 329
Expected indentation of 14 space characters but found 16

File: src/components/userProfile/UserProfile.js [hunk 18]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
+                colors={['#546979', '#08233a']}>
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:329]: The closing bracket must be aligned with the line containing the opening tag (expected column 13 on the next line)

File: src/components/userProfile/UserProfile.js [hunk 19]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
+              <Image
+                  style={styles.profileImage}
+                  source={this.state.profileImage} />
+            </LinearGradient>
+          </Animated.View>
+          <Image
+              style={styles.bookmarkIcon}
+              source={require('../../resources/icon-bookmark.png')}/>
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:337]: 1. A space is required before closing bracket

2. The closing bracket must be aligned with the line containing the opening tag (expected column 11 on the next line)

File: src/components/userProfile/UserProfile.js [hunk 20]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
+            <LinearGradient
+                style={styles.profileImgGradient} start={[0.0, 0.25]} end={[0.5, 1.0]}
+                colors={['#546979', '#08233a']}>
+              <Image
+                  style={styles.profileImage}
+                  source={this.state.profileImage} />
+            </LinearGradient>
+          </Animated.View>
+          <Image
+              style={styles.bookmarkIcon}
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:336]: line 336, 337
Expected indentation of 12 space characters but found 14

File: src/components/userProfile/UserProfile.js [hunk 21]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
-          <Image style={styles.bookmarkIcon}
-            source={require('../../resources/icon-bookmark.png')}/>
+          <Animated.View
+              {...this.panResponder.panHandlers}
+              style={[this.state.pan.getLayout(), imageStyle]}>
+            <LinearGradient
+                style={styles.profileImgGradient} start={[0.0, 0.25]} end={[0.5, 1.0]}
+                colors={['#546979', '#08233a']}>
+              <Image
+                  style={styles.profileImage}
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:331]: line 331, 332
Expected indentation of 16 space characters but found 18

File: src/components/userProfile/UserProfile.js [hunk 22]
```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
...
+                  source={this.state.profileImage} />
```
Comment (MechanicKim) [inline src/components/userProfile/UserProfile.js:332]: The closing bracket must be aligned with the line containing the opening tag (expected column 15 on the next line)

File: src/components/userProfile/UserProfile.js (hunk 6 above)
Comment (heartjigi) [reply src/components/userProfile/UserProfile.js:107]: My JSCS says this right

**Pull request closed without merging**